robot_sf/sim/simulator.py

```
@@ -49,77 +55,115 @@ class Simulator:
             forces = pysf_make_forces(sim, config)
             forces = [f for f in forces if not isinstance(f, ObstacleForce)]
             if self.config.prf_config.is_active:
                 for robot in self.robots:
                     self.config.prf_config.robot_radius = robot.config.radius
                     forces.append(
                         PedRobotForce(
-                            self.config.prf_config,
-                            sim.peds,
-                            lambda: robot.pos
-                            )
+                            self.config.prf_config, sim.peds, lambda: robot.pos
```

Ruff (0.7.0)

Function definition does not bind loop variable `robot`

(B023)